pkg/models/v1/models.go

```diff
@@ -508,6 +508,11 @@ func NewCertificateRequest(vasp *pb.VASP) (certRequest *CertificateRequest, err
 		log.Debug().Str("vasp_id", vasp.Id).Str("certreq_id", certRequest.Id).Msg("location information not found or incomplete")
 	}
 
+	// Update certificate delivery method with the method on the VASP
+	if err = UpdateCertificateRequestDelivery(certRequest, vasp); err != nil {
+		return nil, err
+	}
```

Comment:
I'm hoping that when the webhook/email delivery is on the VASP we can simply add the data after line 485 above without having to jump through the GDS extra hoops. See the comment in the proto file.